lakey_service/account/commands.py

```diff
@@ -12,11 +12,13 @@
     serializers,
     HTTPCommands,
 )
+from lily.base.events import EventFactory
```

Comment:
you don't have to perform direct import from EventFactory since every command inherits from it, see below